src/taipy/core/_entity/_migrate.py

```diff
@@ -492,19 +500,19 @@ def __migrate(entities: Dict, versions: Optional[Dict] = None) -> Tuple[Dict, Op
     return entities, versions
 
 
-def _migrate_fs_entities(path: str) -> None:
```

Comment:
Why removing the returned type?

Reply:
It's just not necessary so I removed it to make it cleaner.

Reply:
Actually it should always have a return type hint. Either `-> None` or `-> NoReturn `. Mypy asks to always keep the return type explicit. Since it didn't break on pre-commit and the GH action, maybe we are ignoring this specific mypy error, in that case we can decide what should be the standard (I prefer to always have a type hint if possible)

Reply:
I do prefer too. It is easier to understand what does a function/method when just reading the signature. It's a way to document it.